### PR TITLE
[12.0][MIG] web_ir_actions_act_view_reload

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@pnajman-modoolar would like to ask for your review, if you don't mind

**Review by pnajman-modoolar — APPROVED** [review]
Thanks for the migration :)
Approved.

**Comment by alexey-pelykh** [comment]
@pnajman-modoolar Thanks for the review! Do you know anyone else to summon who may review as well?

**Comment by pnajman-modoolar** [comment]
@alexey-pelykh You could try with good people who reviewed version 11.0 of this module.
I would look them up in the 11.0 pull request.

**Comment by alexey-pelykh** [comment]
Good people of 11.0 PR (@elicoidal @simahawk @bealdav @zaoral), I summon you to take a look on this PR if you don't mind

File: web_ir_actions_act_view_reload/views/web_ir_actions_act_view_reload.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
```
Comment (elicoidal) [inline web_ir_actions_act_view_reload/views/web_ir_actions_act_view_reload.xml]: remove `# `

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by alexey-pelykh** [comment]
Thanks for the reviews! I'd be glad to help likewise, thus feel free to summon me 👍

File: web_ir_actions_act_view_reload/static/src/js/web_ir_actions_act_view_reload.js [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
+            if (action.type === 'ir.actions.act_view_reload') {
+                return this._executeReloadAction(action, options);
+            }
+
+            return this._super.apply(this, arguments);
+        },
+
+        _executeReloadAction: function (action, options) {
+            if (this.controllers && this.controllerStack && this.controllerStack.length > 0)  {
+                var controller = this.controllers[this.controllerStack[0]]
```
Comment (simahawk) [inline web_ir_actions_act_view_reload/static/src/js/web_ir_actions_act_view_reload.js]: shouldn't you use `getCurrentController`?

**Review by simahawk — CHANGES_REQUESTED** [review]
LG, just a question

File: web_ir_actions_act_view_reload/static/src/js/web_ir_actions_act_view_reload.js (hunk 2 above)
Comment (alexey-pelykh) [reply web_ir_actions_act_view_reload/static/src/js/web_ir_actions_act_view_reload.js]: Good point, haven't found that function, thanks for hint - updated

**Comment by alexey-pelykh** [comment]
@simahawk please re-review

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
IMO Check Jslint

**Review by nikul-serpentcs — APPROVED** [review]
Otherwise Code LGTM

**Comment by alexey-pelykh** [comment]
@nikul-serpentcs is there an automated check for it? I'm not sure how to run it on Odoo

**Comment by nikul-serpentcs** [comment]
@alexey-pelykh Could you please check [here](https://travis-ci.org/OCA/web/jobs/457346440#L870-L873)

**Comment by simahawk** [comment]
@alexey-pelykh LG, can you check the warnings @nikul-serpentcs linked?

**Comment by alexey-pelykh** [comment]
@simahawk @nikul-serpentcs all good now

**Comment by simahawk** [comment]
I still see `web_ir_actions_act_view_reload/static/src/js/web_ir_actions_act_view_reload.js:25: [W7903(javascript-lint), ] Trailing spaces not allowed. [Error/no-trailing-spaces]`

Am I wrong?

Sorry for this boring requirements but if we don't fix them while we here they are going to stay forever and maybe become errors in the future... thanks for your patience! :wink:

**Comment by alexey-pelykh** [comment]
@simahawk sorry, dunno where I've looked exactly yet I can swear this warning wasn't there 🙈fixed!

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@alexey-pelykh thanks :)

**Pull request merged**